storage/log_storage.go

```diff
@@ -136,4 +136,8 @@ type LogRootWriter interface {
 type LogMetadata interface {
 	// GetActiveLogs returns a list of the IDs of all the logs that are configured in storage
 	GetActiveLogIDs(ctx context.Context) ([]int64, error)
+
+	// GetUnsequencedCounts returns a map of the number of unsequenced entries
+	// by log ID.
+	GetUnsequencedCounts(ctx context.Context) (map[int64]int64, error)
```

Comment:
Would it be useful to have `GetUnsequencedCount(ctx, logID)`, either as a replacement or in addition to this method? (Not that I think it would be any better performance-wise.)